### PR TITLE
SElinux from RHEL8 does not overwrite existing labels 

### PR DESCRIPTION
**Pull request by pkesavap**
SElinux from RHEL8 does not overwrite existing labels with "fcontext …-a" , it throws "ValueError: File context for {{mount path}}  already defined"

So a pretask which deletes the label which corresponds to the existing mount points is added

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=1820728

Signed-off-by: Prajith Kesava Prasad <pkesavap@redhat.com>

### CODE REVIEW

File: playbooks/set-selinux-labels.yml [hunk 1]
```diff
@@ -4,6 +4,12 @@
   gather_facts: no
 
   tasks:
+  - name: Deleting existing SELinux label if any
+    shell: semanage fcontext -d {{ item.path }}
+    with_items: "{{ mntpath }}"
+    when: mntpath is defined
+    ignore_errors: yes
```
Comment (gobindadas) [inline playbooks/set-selinux-labels.yml]: ingnoring error? If delete selinux lebel has some issue, which means this task will not mark as failure as error is ignored, then next task will fail right while applying same lebel again?
Why can't simply check if the paricular label is already set then ignore?

Comment (pkesavap) [reply playbooks/set-selinux-labels.yml]: @gobindadas the delete selinux label throws error only if that mount point is not labeled,
(the intial case) i.e when mountpath is not labeled in that case the next task will not fail.

Only that dependence is there, which makes the second task safe.
thats why i implemented it.

**Review by kshithijiyer — CHANGES_REQUESTED** [review]
Squash your commits and resolve merge conflict

**Comment by pkesavap** [comment]
Closing this PR, as this the issue is handled , in another fix .

**Pull request closed without merging**